students/GuyCChan/session09/test_objOriMailRoom.py

```diff
@@ -0,0 +1,226 @@
+from objOriMailRoom import Donor, DonorCollection, make_report, letters_to_donors, test_data
+import pytest
+
+# Tests for Donor class.
+def test_init():
+    db = donors_db()
+    assert len(donors_db) == 0
+
+def test_Donor_in_db():
```

Comment:
make your test names meaningful - this is not testing if there is a donor on the db...